operations/network/customer_accessible_networks/can_to_chn/network/chn_enable.md

```diff
@@ -467,10 +477,20 @@ Administrators should enable CFS for UAN, ensure plays run successfully and then
 
 1. (`ncn-m001#`) Enable CFS changes on UAN.
 
+    ```bash
```

Comment:
The indentation in this step still seems inconsistent (some places it is 3 spaces and other places it is 4).